### ISSUE TITLE
Can not add or remove  supersedence

### ISSUE THREAD

**Issue by AlkHacNar** [MSEndpointMgr/IntuneWin32App#123]
I think the add-intunewin32appsupersedence and remove-intunewin32appsupersedence have the same issue like this one [#122](https://github.com/MSEndpointMgr/IntuneWin32App/issues/122)

for the add-intunewin32appsupersedence i found a workaround, which could also work for dependencies. for the remove-intunewin32appsupersedence I couldn't find one, but I'm not really good with the api. I found [this](https://learn.microsoft.com/en-us/graph/api/intune-apps-mobileappsupersedence-delete?view=graph-rest-beta) but couldn't get it working

`$module = Get-Content "C:\Program Files\WindowsPowerShell\Modules\IntuneWin32App\1.4.2\Public\Add-IntuneWin32AppSupersedence.ps1"
    $module[68] = '                    "relationships" = @()'
    $module[70] = '                if ($Dependencies) { @($Win32AppRelationshipsTable.relationships += $Supersedence; $Win32AppRelationshipsTable.relationships += $Dependencies) } else { @($Win32AppRelationshipsTable.relationships += $Supersedence) }'
    Set-Content "C:\Program Files\WindowsPowerShell\Modules\IntuneWin32App\1.4.2\Public\Add-IntuneWin32AppSupersedence.ps1" $module`

**Comment by PZan** [comment]
I'm seeing the exact same issue here, but your workaround doesn't seem to make any difference for me. Could you share an example of how you're completing a supersedence sequence?
This is what I'm trying to do:
`
$AppSupersedenceObject = New-IntuneWin32AppSupersedence -ID $SupersedeMe.id -SupersedenceType 'Update'
Add-IntuneWin32AppSupersedence -ID $KeepThisApp.id -Supersedence $AppSupersedenceObject
`

But all I get back is:
`
WARNING: An error occurred while configuring supersedence for Win32 app: <app-id>. Error message: BadRequest: The request is malformed or incorrect. Expected array for value of
property: Collection(microsoft.graph.mobileAppRelationship)
`

**Comment by AlkHacNar** [comment]
@NickolajA

I think it's the same solution like in #122. Do you fix it in 1.4.4 too? 😅

**Comment by NickolajA** [comment]
Yeah it will be fixed in 1.4.4. This actually seems to be an honest mistake where it would simply attempt to construct the array by using $() instead of @().
